src/components/elements/query-image.tsx

```diff
@@ -8,6 +8,7 @@ type QueryImageProps = {
     data: ImageDataLike
     height?: string
     width?: string
+    max_width?: string
```

Comment:
Do we still need this?

Reply:
`max_width` is still using in other components like src/components/layout/nav.js